src/components/OrderSummaryPlanSelect.js

```diff
@@ -1,18 +1,62 @@
+import { useState } from 'react';
+import DurationMenu from './DurationMenu';
+
 import icon from '../images/icon-music.svg';
 
 function OrderSummaryPlanSelect () {
 
+    // Tempary test data to test functionality on this component and DurationMenu
+    const planOptions = [
+        {
+            duration: 'Annual',
+            price: 59.99
+        },
+        {
+            duration: 'Monthly',
+            price: 6
+        },
+    ];
+    const [displayDurationMenu, setDisplayDurationMenu] = useState(false);
+    const [selectedOption, setSelectedOption] = useState(0);
+
+    // Switches from displaying the selected option to displaying the DurationMenu
+    const onClickHandler = (e) => {
+        e.preventDefault();
+        toggleDurationMenu();
+    }
+
+    // Handler takes 'duration' property passed up from DurationMenu and makes it the selected option
+    const onSelectHandler = (selected) => {
+        toggleDurationMenu();
+        setSelectedOption(selected === 'Annual' ? 0 : 1);
```

Comment:
A minor point, but for readability you might want to avoid having any conditional statements within function arguments. It might be worth assigning the output of this ternary to a constant, and then passing that in as the argument e.g.

```
const option = (selected === 'Annual' ? 0 : 1);
setSelectedOption(option);
```